packages/camera/camera/README.md

```diff
@@ -141,14 +133,16 @@ class _CameraAppState extends State<CameraApp> {
         return;
       }
       setState(() {});
-    }).catchError((e) {
-      switch (e.code) {
-        case 'CameraAccessDenied':
-          print('User denied camera access.');
-          break;
-        default:
-          print('Handle other errors.');
-          break;
+    }).catchError((dynamic e) {
```

Comment:
Please don't use `dynamic` in modern Dart except where absolutely necessary; it bypasses all type checks. `Object` (or when needed, `Object?`) is almost always the better choice.